### PR TITLE
Update build system to link system libcrypto

### PR DESCRIPTION
**Pull request by xiangquanliu**

### CODE REVIEW

File: psw/ae/aesm_service/Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ include $(TOP_DIR)/buildenv.mk
 
 CPPMICROSERVICES_DIR := $(LINUX_EXTERNAL_DIR)/CppMicroServices
 CPPMICROSERVICES_INSTALL := $(CPPMICROSERVICES_DIR)/local-install
-AESM_CONFIG := -DBUILD_SHARED_LIBS=ON -DUS_BUILD_SHARED_LIBS=ON
+AESM_CONFIG := -DBUILD_SHARED_LIBS=ON -DUS_BUILD_SHARED_LIBS=ON -DUSE_PREBUILT_OPENSSL=1$(USE_PREBUILT_OPENSSL) -DFORTIFY_SOURCE_VAL=$(FORTIFY_SOURCE_VAL)
```
Comment (lzha101) [inline psw/ae/aesm_service/Makefile:37]: Typo here? Looks it should be -DUSE_PREBUILT_OPENSSL=$(USE_PREBUILT_OPENSSL)

File: psw/ae/aesm_service/source/utils/CMakeLists.txt [hunk 2]
```diff
@@ -35,6 +35,13 @@ add_library(utils SHARED ${_srcs}
     "../../../../../common/src/sgx_memset_s.cpp"
     "../../../../../common/src/crypto_cmac_128.cpp")
 
+if(NOT "${USE_PREBUILT_OPENSSL}" STREQUAL "0")
```
Comment (lzha101) [inline psw/ae/aesm_service/source/utils/CMakeLists.txt:38]: Looks some build scripts are also required the similar changes, such as qpl, qcnl, etc.

Comment (xiangquanliu) [reply psw/ae/aesm_service/source/utils/CMakeLists.txt:38]: Confirmed with Xun, seems we can't link prebuilt openssl for qcnl.

Comment (lzha101) [reply psw/ae/aesm_service/source/utils/CMakeLists.txt:38]: It looks not the fact. Actually, right now, qpl and qcnl are linking the prebuilt OpenSSL:
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qpl/CMakeLists.txt#L36
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qcnl/CMakeLists.txt#L37
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qcnl/pck_cert_selection/CMakeLists.txt#L50

The motivation for this PR is to switch to system OpenSSL in the untrusted components for some conditions. I think we need to change these scripts to enable the switching on these components.

Comment (llly) [reply psw/ae/aesm_service/source/utils/CMakeLists.txt:38]: `qcnl` and `qpl` in `aesm_service` can be simply removed. They are built out and packaged in DCAP.

Comment (xiangquanliu) [reply psw/ae/aesm_service/source/utils/CMakeLists.txt:38]: Done

**Review by lzha101 — APPROVED** [review]

**Pull request merged**